docs/atlascli/command/atlas-dataFederation-queryLimits-describe.txt

```diff
@@ -12,20 +12,36 @@ atlas privateEndpoints dataLakes aws list
    :depth: 1
    :class: singlecol
 
-List Data Lake private endpoints for your project.
+Return the details for the specified data federation query limit for your project.
 
-To use this command, you must authenticate with a user account or an API key that has the Project Read Only role.
+To use this command, you must authenticate with a user account or an API key that has the Project Owner role.
 
 Syntax
 ------
 
 .. code-block::
    :caption: Command Syntax
 
-   atlas privateEndpoints dataLakes aws list [options]
+   atlas dataFederation queryLimits describe <name> [options]
 
 .. Code end marker, please don't delete this comment
 
+Arguments
+---------
+
+.. list-table::
+   :header-rows: 1
+   :widths: 20 10 10 60
+
+   * - Name
+     - Type
+     - Required
+     - Description
+   * - name
+     - string
+     - true
+     - Identifier of the data federation query limit
```

Comment:
I'll stop calling out these periods - I think it applies to all new "name" descriptions 
```suggestion
     - Identifier of the data federation query limit.
```